frontends/mit-learn/src/page-components/TestimonialDisplay/TestimonialDisplay.tsx

```diff
@@ -75,7 +81,8 @@ const QuoteBody = styled.div(({ theme }) => ({
   },
 }))
 
-const AttestationBlock = styled.div(({ theme }) => ({
+const AttestationBlock = styled.blockquote(({ theme }) => ({
```

Comment:
The blockquote / cite tags are more semantic and unrelated to the rest of the changes in this PR.

I had been hoping that my screenreader would announce the quote (and citation) with these tags. Voiceover does not. So sad.

But JAWS and NVDA (which account for something like 85% of the screenreader usage—in US? worldwide? I'm not sure—do announce semantic quote blocks. So decided to keep it.

See https://adrianroselli.com/2023/07/blockquotes-in-screen-readers.html